viewing_party/main.py

```diff
@@ -0,0 +1,200 @@
+#--------FUNCTIONS FOR WAVE01--------#
+
+def create_movie(movie_title, genre, rating):
+    #If movie title, genre or rating are None, then return None
+    if not movie_title or not genre or not rating:
+        return None
+    #Create a movie dictionary and return it
+    movie_dictionary = {
+        "title" : movie_title,
+        "genre" : genre,
+        "rating" : rating
+    }
+    return movie_dictionary
+
+def add_to_watched(user_data, movie):
+    #User adds a movie to the list value of "watched" key.
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    #User adds a movie to the list value of "watchlist" key.
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    #Watchlist key value (list), will be stored in watchlist_list 
+    # and then iterated.
+    watchlist_list = user_data["watchlist"]
+    #Each element of that list is a movie represented with a 
+    # dictionary containing the keys: title, genre and rating. 
+    for movie in watchlist_list:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            watchlist_list.remove(movie)
+    return user_data
+
+
+#--------FUNCTIONS FOR WAVE02--------#
+
+def get_watched_avg_rating(user_data):
+    #Initialize a counter in 0
+    movies_rating = 0
+    movies_watched_list = user_data["watched"]
+    #If watched movies list is empty, return 0.0 as average
+    if len(movies_watched_list) == 0:
+        return 0.0
+    #If not empty, add each movie rating to the total (movies_rating)
+    for movie in movies_watched_list:
+        movies_rating += movie["rating"]
+    return movies_rating / len(movies_watched_list)
```

Comment:
this works perfectly! Let's see if we can use only one `return` statement

```suggestion
    movies_rating = 0.0
    movies_watched_list = user_data["watched"]
    #If watched movies list is empty, return 0.0 as average
    if len(movies_watched_list) > 0:
        for movie in movies_watched_list:
            movies_rating += movie["rating"]
        movies_rating = movies_rating / len(movies_watched_list)
    return movies_rating
```